tests/eosio.system_tester.hpp

```diff
@@ -788,6 +769,20 @@ class eosio_system_tester : public TESTER {
                                 );
    }
 
+   void issue_and_transfer( const name& to, const asset& amount, const name& manager = config::system_account_name ) {
+      base_tester::push_action( N(eosio.token), N(issue), manager, mutable_variant_object()
```

Comment:
Group into a single transaction as it was previously in `issue`. Otherwise, there is a higher likelihood of unintended duplicate transaction errors.

Reply:
Done.